drivers/docker/util/stats_posix.go

```diff
@@ -12,17 +12,28 @@ import (
 
 var (
 	DockerMeasuredCPUStats = []string{"Throttled Periods", "Throttled Time", "Percent"}
-	DockerMeasuredMemStats = []string{"RSS", "Cache", "Swap", "Usage", "Max Usage"}
+
+	// cgroup-v2 only exposes a subset of memory stats
+	DockerCgroupV1MeasuredMemStats = []string{"RSS", "Cache", "Swap", "Usage", "Max Usage"}
+	DockerCgroupV2MeasuredMemStats = []string{"Cache", "Swap", "Usage"}
```

Comment:
In https://github.com/hashicorp/nomad/issues/9073 we have an issue where metrics that aren't being measured are still being emitted to the metrics endpoint. Does this help with that situation here or will this still result in empty metrics being emitted? See [`task_runner.go#L1309-L1336`](https://github.com/hashicorp/nomad/blob/v1.0.4/client/allocrunner/taskrunner/task_runner.go#L1309-L1336).

Reply:
This wouldn't address that issue.  We can update task_runners.go to see if the metrics are measured.